inc/namespace.php

```diff
@@ -38,6 +38,10 @@ function bootstrap( Module $module ) {
 		add_action( 'muplugins_loaded', __NAMESPACE__ . '\\Site_Verification\\bootstrap' );
 	}
 
+	if ( defined( 'TACHYON_URL' ) && TACHYON_URL ) {
+		add_action( 'muplugins_loaded', __NAMESPACE__ . '\\use_tachyon_img_in_metadata' );
+	}
+
```

Comment:
@roborourke is this sufficient to meet this requirement?

> Filters are not applied if tachyon is disabled in config

`$settings` here are for `altis.modules.seo`. As far as I can tell `tachyon` is enabled when either of the modules are enabled `altis.modules.media.tachyon` or `altis.modules.media.smart-media`. So I decided to test for `TACHYON_URL` directly rather per variety of settings.

Reply:
Tachyon is enabled if `altis.modules.media.tachyon` is `true` - the smart media one looks at that setting to decide if it should load or not because it depends on Tachyon. So you only need to look at that. `TACHYON_URL` can be defined even if the tachyon plugin isn't loaded so you need to change this to look at the config setting.

Reply:
You can check the global config using `Altis\get_config()['modules']['media']['tachyon'] ?? false`

Reply:
Thank you! My bad for overlooking it.